corehq/apps/consumer_user/templates/change_password.html

```diff
@@ -0,0 +1,63 @@
+{% extends "consumer_user_base.html" %}
```

Comment:
we  noticed that these files aren't in the regular django template path. Could you move them `corehq/apps/consumer_user/templates/change_password.html` instead of `corehq/apps/consumer_user/templates/consumer_user/change_password.html`. Think you could move them all? You'll also need to replace all the extends.

Reply:
Moved the templates as per suggestion